src/app/components/resources/resources.component.ts

```diff
@@ -51,6 +120,38 @@ export class ResourcesComponent implements OnInit {
     modalRef.result.then(() => this.loadResources(), console.log);
   }
 
+  toggleResources = function (
+    category: string,
+    optionId: string | number,
+  ): void {
+    const updatedFilters =
+      this.filters[category] && this.filters[category].includes(optionId)
```

Comment:
What do you think about changing the branches to be `this.filters[category] ? this.filters[category].includes(optionId) ? branch1 : branch2 : branch3`? I think that would be a little easier to follow.